common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -1751,14 +1751,20 @@
   "screens.wynntils.characterSelection.name": "Wynntils Custom Character Selection",
   "screens.wynntils.characterSelection.playButton.description": "You can also double click to play",
   "screens.wynntils.characterSelection.playButton.play": "[>] Play with the selected character",
+  "screens.wynntils.chatTabsGui.add": "Add",
+  "screens.wynntils.chatTabsGui.addAndClose": "Add and Close",
   "screens.wynntils.chatTabsGui.autoCommand": "Auto Command",
   "screens.wynntils.chatTabsGui.cancel": "Cancel",
   "screens.wynntils.chatTabsGui.consuming": "Consuming",
+  "screens.wynntils.chatTabsGui.create": "Create New Tab",
   "screens.wynntils.chatTabsGui.delete": "Delete",
+  "screens.wynntils.chatTabsGui.edit": "Editing Tab ",
```

Just a small nit, but you could use format strings.

```suggestion
  "screens.wynntils.chatTabsGui.edit": "Editing Tab %s",
```

and pass the string in the appropriate i18n.get or component.translatable call